scripts/prepare-release.mjs

```diff
@@ -0,0 +1,350 @@
+	console.log( error.message );
+	console.log( error.stack );
+
+	const { confirmation } = await inquirer.prompt( {
+		type: 'confirm',
+		name: 'confirmation',
+		message: 'Roll back and delete release branch?',
+		default: true,
+	} );
+
+	if ( confirmation ) {
+		revertOnError( originalBranchName, releaseBranch );
+	}
+}
+
+/*
+ * HELPER FUNCTIONS FROM HERE.
+ */
+
+/**
+ * Get plugin slug from command arguments.
+ * Throws an error if invalid.
+ *
+ * @return {string} The plugin slug.
+ */
+function getPluginSlug() {
+	const slug = process.argv[ 2 ];
+	if ( ! ( slug in PLUGINS ) ) {
+		throw new Error(
+			'Please provide a valid plugin slug as the first parameter: ' +
+			Object.keys( PLUGINS ).join( ', ' ),
+		);
+	}
+	return slug;
+}
+
+/**
+ * Return file contents given the filepath.
+ * This method will throw an error if the file does not exist.
+ *
+ * @param {string} filepath The file path to be read.
+ * @return {string} The file contents.
+ */
+function readFileContents( filepath ) {
+	let contents = false;
+	try {
+		contents = fs.readFileSync( filepath, 'utf8' );
+	} catch ( err ) {
+		throw new Error( `File (${ filepath }) could not be read.` );
+	}
+
+	return contents;
+}
+
+/**
+ * Ask for confirmation on new version and dependency versions through the CLI.
+ *
+ * @param {string} newVersion   The new version.
+ * @param {string} fileContents The current contents of the main plugin file.
+ * @return {boolean} Whether the confirmation was accepted or not.
+ */
+async function askForConfirmation(
+	newVersion,
+	fileContents,
+) {
+	// WP Versions
+	const currentWPRequiresAtLeast = fileContents.match(
+		/Requires at least: (.*)/,
+	)[ 1 ];
+	const currentWPTestedUpTo      = fileContents.match( /Tested up to: (.*)/ )[ 1 ];
+	// PHP
+	const currentRequiresPhp       = fileContents.match( /Requires PHP: (.*)/ )[ 1 ];
+
+	// Display all versioning information and ask for confirmation.
+	console.log( `🚀 Preparing new release:`, chalk.bold( `${ pluginSlug } ${ newVersion }` ) );
+	console.log( `-----------------------------` );
+	console.log( chalk.bold( '📦 Plugin header:' ) );
+	console.log( `   Version:`, chalk.bold.green( newVersion ) );
+	console.log( `   (WP)  Requires at least:`, chalk.bold( currentWPRequiresAtLeast ) );
+	console.log( `   (WP)  Tested up to:`, chalk.bold( currentWPTestedUpTo ) );
+	console.log( `   (PHP) Requires PHP:`, chalk.bold( currentRequiresPhp ) );
+	console.log( `-----------------------------` );
+	console.log( `ℹ️️  Make sure a ` + chalk.bold( `milestone ${ newVersion }` ) + ` exists GitHub, and all PRs are assigned to the milestone.` );
+	console.log( `-----------------------------` );
+	console.log( `ℹ️️  Make sure you are logged in to GH CLI with \`gh auth login\`.` );
+	execSync( 'gh auth status' );
+	console.log( `-----------------------------` );
+	console.log( `Pull requests to include (milestone ${ newVersion }):` );
+
+	execSync( ghPrs, { stdio: 'inherit' } );
```

Comment:
Maybe I miss something but I think this doesn't do anything

Reply:
This one should just output the list of PRs directly to the console.